.changeset/four-windows-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": minor
+---
+
+Removed `FileDropZone` from Lab and promoted it to Core.
```

Comment:
```suggestion
Removed `FileDropZone` from Lab and promoted it to core.
```

Reply:
do we have a consensus about it? I see both being used in the release notes, but this makes sense given how we talk about them in the site... updated :)